src/src/rx-serial/SerialSatellite.cpp

```diff
@@ -0,0 +1,126 @@
+#include "SerialSatellite.h"
+#include "CRSF.h"
+#include "config.h"
+#include "device.h"
+
+#if defined(TARGET_RX)
+
+extern RxConfig config;
+
+static constexpr uint8_t CRSF_TO_SATELLITE_CH_MAP[12] = {
+    // clang-format off
+    // CRSF         // Satellite
+    /*Channel 0*/   1,
+    /*Channel 1*/   2,
+    /*Channel 2*/   0,
+    /*Channel 3*/   3,
+    /*Channel 4*/   11,
```

Comment:
Why is channel 5 mapped to 11?

Reply:
Many FCs that use satellite receivers use the 5th channel for various things like flight mode selection, pitch for helicopters etc. A 2 position switch in channel 5 may not be enough for many users, especially when it is tied to arming. I mapped channel 5 to 11 at the output to give users as many general purpose outputs as possible at the lower channels. However, as I indicated in my very first message, I am open to suggestions related to mapping, since as @pkendall64 pointed out, mapping would be useful for SBUS too.